### PR TITLE
wip, fix component example

### PR DESCRIPTION
**Pull request by StuartHarris**

### CODE REVIEW

File: crates/wasm-compose/example/middleware/Cargo.lock
Comment (charypar) [inline crates/wasm-compose/example/middleware/Cargo.lock:1]: Don't think this was committed originally.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "cmake.configureOnOpen": false,
+  "rust-analyzer.linkedProjects": [
+    "./crates/wasm-compose/example/server/Cargo.toml"
+  ]
+}
```
Comment (charypar) [inline .vscode/settings.json]: Probably shouldn't commit this

**Comment by charypar** [comment]
This now compiles and works with the service component. Haven't yet tested the composition.

File: crates/wasm-compose/example/middleware/Cargo.lock
Comment (StuartHarris) [reply crates/wasm-compose/example/middleware/Cargo.lock:1]: probably should be for examples

**Pull request merged**